.env.example

```
@@ -2,5 +2,6 @@ DATABASE_URL="sqlite:schwab.db"
 WS_RPC_URL=wss://arbitrum-one-rpc.publicnode.com
 ORDERBOOK=0xf6fdcb30fd027c73cbacd54e22f02987e3116ebc
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ORDERBOOK key should go before the WS_RPC_URL key

(UnorderedKey)